src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -448,12 +288,12 @@ void LedgerConsensusImp::mapCompleteInternal (
             << "Not creating disputes: identical position.";
     }
 
-    mAcquired[hash] = map;
+    acquired_[hash] = map;
 
     // Adjust tracking for each peer that takes this position
     std::vector<NodeID> peers;
     auto const mapHash = map->getHash ().as_uint256();
-    for (auto& it : mPeerPositions)
+    for (auto& it : peerPositions_)
```

Comment:
while renaming, let's clear up these warnings (there are two more, at lines 1413 & 1482, I'll comment them) 
Warning	C6246	Local declaration of 'it' hides declaration of the same name in outer scope. For additional information, see previous declaration at line '237'